ocpmodels/trainers/base_trainer.py

```diff
@@ -328,11 +328,11 @@ def load_model(self):
         if self.logger is not None:
             self.logger.watch(self.model)
 
-        self.model = OCPDataParallel(
-            self.model,
-            output_device=self.device,
-            num_gpus=1 if not self.cpu else 0,
-        )
+        #        self.model = OCPDataParallel(
+        #            self.model,
+        #            output_device=self.device,
+        #            num_gpus=1 if not self.cpu else 0,
+        #        )
```

Comment:
Is this meant to be commented out?

Reply:
yeah, this PR only had the pipeline parallel part (without DDP).